app/controllers/stops_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
 
   def show
     @stop=Stop.find(params[:id])
+    @review=@stop.reviews
     render "static_views/index"
```

Comment:
I dont think you are really using either of these instance variables either because the show page is in react. Remove them